armi/apps.py

```diff
@@ -153,7 +153,9 @@ def getSettings(self) -> Dict[str, Setting]:
                         defaultsCache[pluginSetting.settingName] = pluginSetting
                 else:
                     raise TypeError(
-                        f"Invalid setting definition found: {pluginSetting}"
+                        "Invalid setting definition found: {} ({})".format(
+                            pluginSetting, type(pluginSetting)
```

Comment:
I kind of prefer the f-string. 
```
f"Invalid setting definition found: {pluginSetting}\n"
f"({type(pluginSetting)})"
```

Reply:
It wasn't working for some reason... didn't want to spend too much time trying to figure it out.

Reply:
I bet you forgot the second `f`